app/src/main/java/io/gnosis/safe/ui/transactions/details/SigningOwnerSelectionFragment.kt

```diff
@@ -66,9 +66,19 @@ class SigningOwnerSelectionFragment : BaseViewBindingFragment<FragmentSigningOwn
             backButton.setOnClickListener {
                 findNavController().navigateUp()
             }
-            if (!isConfirmation) {
-                description.setText(R.string.signing_owner_selection_list_rejection_description)
+
+            when(signingMode) {
+                SigningMode.CONFIRMATION -> {
+
```

Comment:
Does it make sense for the sake of consistency to set the description explicitly, here? Although it is already set in the layout.